R/logs.R

```diff
@@ -0,0 +1,47 @@
+library(log4r)
+
+read_environment_variable <- function(name, default_value) {
+  v <- Sys.getenv(name)
+  if (is.na(v) || v == "") {
+    v <- default_value
+  }
+  return(v)
+}
+
+log_file_name <- read_environment_variable(
+  "CAP_SEURAT_CONVERTER_LOG_FILE_NAME",
+  paste(getwd(), "logfile.txt",
+    sep = "/src/"
+  )
+)
+log_threshold <- read_environment_variable(
+  "CAP_SEURAT_CONVERTER_LOG_LEVEL",
+  "DEBUG"
+)
```

Comment:
Don't need to do that in the basic package, we would control in on the server side

Reply:
done